doc/distributions/nc_chi_squared.qbk

```diff
@@ -72,7 +72,7 @@ for different values of [lambda]:
 
       non_central_chi_squared_distribution(RealType v, RealType lambda);
 
-Constructs a Chi-Squared distribution with /v/ degrees of freedom
+Constructs a Chi-Squared distribution with v degrees of freedom
```

Comment:
This one is incorrect, the /v/ will put it in italics.

Reply:
I would replace it by [nu]

Reply:
I have made the change. 
but why do we have to write [nu] in square brackets
in the latex link, you sent  \nu  will give the correct.

you can point me to some resource link